snewpdag/plugins/SeriesBinner.py

```diff
@@ -32,12 +32,13 @@
 from snewpdag.dag import Node
 
 class SeriesBinner(Node):
-  def __init__(self, in_field, nbins, xlow, xhigh,
+  def __init__(self, in_field, field2, nbins, xlow, xhigh,
```

Comment:
Could we change "field2" to "in_field2" to follow the style of the first input field argument?  I suspect it should also be an optional field - but perhaps a slightly more mysterious question:  how is this used?  I see self.field2 used only in assignments in this plugin, and in other plugins there's no reference I can find to the "field2" key (which perhaps you were trying to use to tell downstream plugins about using t_true, but in which case I'd put the indirection argument in those plugins to tell them directly, rather than through the payload)

Reply:
Sure, I can change that. I actually did not know which was the best name/best way of implementing.
So, if you think it is better to do it in another way, feel free to modify, but I did not find a better way of giving an information from the generator to the calculation (in that case TimeDistDiff).
Basically, in order to build the time uncertainty (true simulated time delay - time delay estimated from the algorithm), I needed to propagate the true time delays from the generator for each experiment to the next steps.

Reply:
The changes have been applied in the last commit of the branch